README.md

```diff
@@ -418,6 +421,8 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
 - <a name="Mondal21iccvw"></a>Mondal, A., Shashant, R., Giraldo, J. H., Bouwmans, T., Chowdhury, A. S.,  
 *[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/papers/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.pdf)*,  
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [Youtube](https://youtu.be/ST6Z-3SlNS4), [Code](https://github.com/anindya2001/GSCEventMOD).
+- [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
```

Comment:
For some reason this link does not show up in the rich difference view of the pull request. I will check right after merging.